tests/conftest.py

```diff
@@ -17,7 +17,9 @@ def pytest_addoption(parser):
         _e2e_pytest_addoption(parser)
     parser.addoption('--dataset-definitions', action='store', default=None,
                      help='Path to the dataset_definitions.yml file for tests that require datasets.')
```

Comment:
where is it?

Reply:
It is used in `dataset_definitions_fx` fixture in `tests/test_ote_training.py`

Reply:
How can it be tested locally? Especially for external user who does not have our datasets?

Reply:
@LeonidBeynenson Could you please skip these tests if command line parameter "--dataset-definitions" is not set?

Reply:
Fixed.
Now if `--dataset-definitions` parameter is absent, all the tests that depend on them are skipped.